File: Example Code/01 Variables, Data Types, Input and Output/Math_AreaCalculator.cpp
#include <iostream>
using namespace std;

int main()
{
    float width, length;
    cout << "AREA CALCULATOR" << endl;
    
    cout << "Enter width: ";
    cin >> width;
    
    cout << "Enter length: ";
    cin >> length;
    
    float area = width * length;
    
    cout << "Area: " << area << endl;    
    
    return 0;
}

File: Example Code/01 Variables, Data Types, Input and Output/source/Math_AreaCalculator.java
import java.util.Scanner;

public class Math_AreaCalculator
{
    public static void main( String args[] )
    {
        Scanner scanner = new Scanner( System.in );
        double width, length;
        
        System.out.println( "AREA CALCULATOR" );
        System.out.print( "Enter the width and length: " );
        width = scanner.nextDouble();
        length = scanner.nextDouble();
        
        double area = width * length;
        
        System.out.println( "Area: " + area );
    }
}
